services/base-service/models/middleware.js

```diff
@@ -64,9 +64,11 @@ async function convertIds(req, res, next) {
 
     if (user && user?.attributes?.expertId) {
       const expertId = Array.isArray(user.attributes.expertId) ? user.attributes.expertId[0] : user.attributes.expertId;
-      req.query.expertIds.push(expertId);
+      experts.push(`expert/${expertId}`);
     }
+    req.query.expert=experts.join(',');
```

Comment:
This converts the sitefarm potential ids to a comma separated string of experts.  It's returned that way to match our other tools that allow `expert=expert/FOOB,expert/BARfar` as a query parameter